src/build/arg/mod.rs

```diff
@@ -4132,6 +4127,23 @@ impl<'help> Arg<'help> {
     }
 }
 
+impl<'a> Arg<'a> {
+    pub(crate) fn _debug_asserts(&self) -> bool {
```

Comment:
Shouldn't return `bool` either.

Reply:
Same reason as above. We are doing `assert!(arg._debug_asserts)`.